src/server/routes/collection.js

```diff
@@ -112,6 +253,11 @@ router.post('/:collectionId/add', auth.isAuthenticated, auth.isCollectionOwnerOr
 					bbid,
 					collectionId: collection.id
 				}).save(null, {method: 'insert'});
+				const collectionModel = await new UserCollection({id: collection.id}).fetch({
+					require: true
+				});
+				collectionModel.set('last_modified', new Date());
+				await collectionModel.save(null, {method: 'update'});
```

Comment:
Here we can do that in one query. [From the docs](https://bookshelfjs.org/api.html#Model-instance-save):
```
// UPDATE authors SET "bio" = 'Short user bio' WHERE "id" = 1
new Author({id: 1, first_name: 'User'})
  .save({bio: 'Short user bio'}, {patch: true})
  .then((model) => {
    // ...
  })
```